src/transform.ts

```diff
@@ -609,50 +609,50 @@ export interface LoessTransform {
 }
 
 export function isLoess(t: Transform): t is LoessTransform {
-  return t['loess'] !== undefined;
+  return 'loess' in t;
 }
 
 export function isSample(t: Transform): t is SampleTransform {
-  return t['sample'] !== undefined;
+  return 'sample' in t;
 }
 
 export function isWindow(t: Transform): t is WindowTransform {
-  return t['window'] !== undefined;
+  return 'window' in t;
 }
 
 export function isJoinAggregate(t: Transform): t is JoinAggregateTransform {
-  return t['joinaggregate'] !== undefined;
+  return 'joinaggregate' in t;
 }
 
 export function isFlatten(t: Transform): t is FlattenTransform {
-  return t['flatten'] !== undefined;
+  return 'flatten' in t;
 }
 export function isCalculate(t: Transform): t is CalculateTransform {
-  return t['calculate'] !== undefined;
+  return 'calculate' in t;
 }
 
 export function isBin(t: Transform): t is BinTransform {
-  return !!t['bin'];
```

Comment:
What if `bin: false`? We should not change this one here.

Reply:
bin's type is `true | BinParams`. 

If we worry about falsy for the primary properties, then maybe we shouldn't change this for all transforms?

For example, 

`{aggregate: null/undefined}` is probably not a reasonable AggregateTransform?

Reply:
Bin is the only one I'm really worried about since people may expect that `false` works since we support `true`.

Reply:
But why would you do that in transform? 
If so, does that make it a bin transform or not? Perhaps an inactive bin transform? So “in” is still correct.

Reply:
Ohh, nm, I looked wrong.